dependency/catalog_services.go

```diff
@@ -120,10 +118,22 @@ func (d *CatalogServicesQuery) CanShare() bool {
 
 // String returns the human-friendly version of this dependency.
 func (d *CatalogServicesQuery) String() string {
+	var name string
 	if d.dc != "" {
-		return fmt.Sprintf("catalog.services(@%s)", d.dc)
+		name = name + "@" + d.dc
```

Comment:
NIT: Better to use fmt.Sprintf here and the next few places.
```suggestion
		name = fmt.Sprintf("%s@%s", name, d.dc)
```

Reply:
`fmt.Sprintf` is generally used when you need specific format, here we're just concatenating and using `fmt.Sprintf` would be overkill (same pattern in org's codes).